### DISCUSSION TITLE
Point to configuration file on startup?

### DISCUSSION THREAD

**Discussion by kenske** [tolgee/tolgee-platform#1261] [Q&A]
For various reasons, instead of loading the config options as environment variables, it would be easier for me to mount an .env file with the configuration inside the container, and then have Tolgee use it on startup. Is there a parameter that would make this possible?

**Comment by JanCizmar** [comment]
Hello, why not to use .env file with `--env-file` attribute of `docker run` or `env_file` of docker compose?

https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option

**Reply by kenske** [reply]
I'm not using docker compose, I'm using Kubernetes. My current setup includes a controller that fetches secrets (for the DB credentials, etc) from an external secrets manager. It's easier to store the result in a file and mount it as a volume in the container than to load the contents as environment variables. Ideally I would like a way to set a environment variable with the path to the config file, and then Tolgee would load the rest of the config from here.  This would also solve the issue of needing to replace dots and underscores in configuration keys.

**Reply by JanCizmar** [reply]
OK, we are using Spring framework on backend so you can probably just set this env variable: 
`spring_config_location=/data/config.properties`

so it will use the config provided in /data/config.properties

Try that.
